kong/reports.lua

```diff
@@ -143,6 +142,15 @@ local function send_report(signal_type, t, host, port)
   if not ok then
     return
   end
+  local opts = {
+    ssl_verify = false,
+  }
+  local hs_ok, err = sock:tlshandshake(opts)
+  if not hs_ok then
+    local msg = "Failed to complete TLS handshake" .. err
+    log(WARN, msg)
+    return msg
```

Comment:
```suggestion
    log(DEBUG, "failed to complete TLS handshake for reports: ", err)
    return
```

Also add `local DEBUG = ngx.DEBUG` above.

I don't think we need to be that noisy when phone home fails as there is no actions the user can take in those case. Just ignore the errors and return should be sufficient.